department-of-veterans-affairs/va.gov-cms
# VACMS-5644: Remove general access to taxonomy overview.

## Description

Resolves #5644 

Non content-admins don't need access to taxonomy overview pages. 

## Testing done

phpunit, manual

## Screenshots

Before 

![Witt_Cook_va_gov___VA_gov_CMS](https://user-images.githubusercontent.com/643678/122567995-45df0100-d017-11eb-9143-e843118a13fc.png)


After

![Witt_Cook_va_gov___VA_gov_CMS](https://user-images.githubusercontent.com/643678/122568147-76269f80-d017-11eb-9be0-53019ccaa287.png)


## QA steps

As a content_publisher such as vanessa.luxen@va.gov, login. You should no longer see the Structure > Taxonomy in the toolbar. 

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
